packages/dashboard/src/pages/Settings/regexp.ts

```diff
@@ -0,0 +1,5 @@
+export const passwordRegexp = /^(?=.{8,20}$)(?=.*[A-Z])(?=.*[a-z])(?=.*[0-9])(?=.*[^A-Za-z0-9]).*/
+
+export const emailRegexp = /^[\w-\.]+@([\w-]+\.)+[\w-]{2,4}$/
```

Comment:
[Stop Validating Email Addresses With Regex](https://davidcel.is/posts/stop-validating-email-addresses-with-regex/). If you need to check email address, please use `<input type="email" />`